administrator/components/com_templates/tmpl/template/default_updated_files.php

```diff
@@ -57,13 +57,15 @@
 									<a class="hasTooltip" href="<?php echo Route::_('index.php?option=com_templates&view=template&id=' . (int) $value->extension_id . '&file=' . $value->hash_id); ?>" title="<?php echo Text::_('JACTION_EDIT'); ?>"><?php echo base64_decode($value->hash_id); ?></a>
 								</td>
 								<td>
-									<?php echo $value->created_date; ?>
+									<?php $created_date = $value->created_date; ?>
+									<?php echo $created_date > 0 ? HTMLHelper::_('date', $created_date, 'Y-m-d H:i:s') : '-'; ?>
```

Comment:
There are language constants for date formats. I'm not sure but the custom field is using them.